CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+Version 0.3.0 (in development)
```

Comment:
I know we've already planned for a 0.3.0 release, but for the purposes of this heading, should we avoid referencing version numbers on the off-chance that they change? For example, if we released v1.6 and decided that the next version is tentatively 1.6.1 (in development), but later decide it needs to be 1.7, we'd be misleading people about the existence of a 1.6.1 release. Maybe we can just not specify it here?